docs/content/en/docs/troubleshooting/_index.md

```diff
@@ -0,0 +1,30 @@
+---
+title: "Troubleshooting"
+weight: 8
+icon: "faq"
+---
+
+## Reporting a problem
+
+Before you report a problem, make sure to retrieve the necessary information from your cluster.
+
+### Automatic log & state collection
+
+You collect information in a Kubernetes cluster using the Cilium CLI:
+
+```shell-session
+$ cilium-cli sysdump
```

Comment:
We should put a note at least how to install cilium-cli here. People might not be aware.

Reply:
It's  in the link below. There are many details on this so I thought it would be best to refer to the Cilium docs for this. Thoughts?